### ISSUE TITLE
Social Preview Cards shall show actual page preview instead of root page preview

### ISSUE THREAD

**Issue by dialogik** [dialogik-tv/dialogiktv-frontend#2]
When you publish a link to a specific dialogik.tv Tools page on Twitter, Discord, Facebook etc., a link preview is shown. Currently, only basic page information is previewed, but not a page specific preview.

The preview shall show the proper page title, page description and - if possible - a page specific picture/image.

**Comment by dialogik** [comment]
C.f. https://stackoverflow.com/questions/64713585/social-open-graph-attributes-not-working-in-my-nuxt-application

**Comment by dialogik** [comment]
I ran a couple of approaches that I hoped would fix the problem but none of them seemed to have helped.

However, just now [I posted a Tweet](https://twitter.com/dialogikTV/status/1375415066816831488?s=20) and – *quel surprise!* – it properly displays the Tool preview (well just title and body since there are no icon uploads on dialogik.tv).

![image](https://user-images.githubusercontent.com/1260548/112627854-6d686800-8e32-11eb-820b-7eabba8dfbca.png)

It would be interesting to know what exactly fixed the problem but since I don't have the time to do any forensics right now I'll simply close this issue as completed. Feels good 😃
